plugins/parodos/src/hooks/workflowsPayload.ts

```diff
@@ -26,20 +26,25 @@ export function walkWorks(
     const next: Work = {
       workName: work.name,
       type: work.workType,
-      arguments: Object.keys(work.parameters ?? {}).map(key => {
-        const value = get(
-          formData,
-          `${prefix === '' ? prefix : `${prefix}.works[${index}]`}${
-            work.name
-          }.${key}`,
-          null,
-        );
+      arguments: Object.keys(work.parameters ?? {})
+        .map(key => {
+          const value = get(
+            formData,
+            `${prefix === '' ? prefix : `${prefix}.works[${index}]`}${
+              work.name
+            }.${key}`,
+            null,
+          );
 
-        return {
-          key,
-          value,
-        };
-      }),
+          if (work.parameters?.[key].required) {
```

Comment:
why only  required parameters?

Reply:
Yeah, I forgot to check `value !== null`

Reply:
But actually in `projectAccessRequestWorkFlow` parameter `ROLE` is optional but at the same time it can't be null, because I get an error on backend

Reply:
the code will exclude all non-required fields though, which is not what we want

Reply:
sending null is fine in up until now, so I would definitely query that.  I'm just concerned this might break things in other places

Reply:
@RichardW98 @anludke sending null for an optional field is causing an error (see above).  Is that a backend bug.  Sending null is fine for other API calls